### PR TITLE
Creative Cloud Test Scripts added

### PR DESCRIPTION
**Pull request by PavanKumarN8**
Hi @sigadamvenkata , Creative Cloud U-NAV Test Scripts added.

### CODE REVIEW

**Comment by PavanKumarN8** [comment]
image

File: features/feds/prodSanity/creativeCloudUnavSanity.spec.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
+module.exports = {
+    name: 'Creative Cloud UNAV Page',
```
Comment (JackySun9) [inline features/feds/prodSanity/creativeCloudUnavSanity.spec.js:2]: please fix all eslint errors

**Comment by JackySun9** [comment]
one more eslint error, please fix

**Review by skumar09 — CHANGES_REQUESTED** [review]
Please fix eslint errors.

cc: @sigadamvenkata

**Comment by PavanKumarN8** [comment]
Hi, @JackySun9, @skumar09, eslint is not accepting json file type.
image
Please assist with solving this error.
cc: @sigadamvenkata

**Comment by PavanKumarN8** [comment]
Hi, @JackySun9, @skumar09 all eslint errors fixed.

cc: @sigadamvenkata

**Review by JackySun9 — APPROVED** [review]

**Review by sigadamvenkata — APPROVED** [review]

**Review by skumar09 — APPROVED** [review]

**Pull request merged**